test/sim/items/sitrusberry.js

```diff
@@ -41,15 +41,23 @@ describe('Sitrus Berry', function () {
 		assert.equal(battle.p1.active[0].hp, 1);
 	});
 
-	it.skip(`should not heal 25% HP if a confusion self-hit would bring the user into Berry trigger range`, function () {
-		battle = common.createBattle([[
+	it(`should not heal 25% HP if a confusion self-hit would bring the user into Berry trigger range`, function () {
```

Comment:
Can you add comments throughout this test when the Deoxys is hitting itself in confusion? It's not clear now.